### DISCUSSION TITLE
Manually self-set account as untracked

### DISCUSSION THREAD

**Discussion by LuisMayo** [RetroAchievements/RAWeb#648] [Ideas]
Hi!
There could be a way of flagging your own account as untracked. This will be useful for people that plays single player games in coop, or share accounts, or want to cheat, or give themselves achievements that got glitched, etc.

This option should have a warning pop-up(or several) in order to avoid it from being used by mistake. 
(Optional) In case of mistake, it may be possible for an admin to revert it if no achievements were obtained since the status was granted.

I'm aware you currently can ask an admin to do it, but this'll easier for everyone implicated and you wouldn't need to "disturb" an admin to flag your own account.

Regards.

**Comment by StingX2** [comment]
The only way this would work is a permanent setting, no on or off switch

**Comment by LuisMayo** [comment]
Sure that's what I was thinking of sorry if I didn't specify it properly.

The only reverting method I suggested was asking to an admin, and only if no new achievements were granted since the change (providing that's possible, I don't know a lot about your system to say). Reverting could not be done by an standard user in any case.

**Comment by ScottFromDerby** [comment]
The original purpose was to basically shadowban cheaters, mainly so they can continue cheating etc without it actually affecting legit players. Not sure about it being a player-facing feature

**Comment by pinguupinguu** [comment]
> Sure that's what I was thinking of sorry if I didn't specify it properly.
> 
> The only reverting method I suggested was asking to an admin, and only if no new achievements were granted since the change (providing that's possible, I don't know a lot about your system to say). Reverting could not be done by an standard user in any case.

Reverting would only be possible if a user resets the entirety of their profile/progress made. This I believe is usually standard procedure for people who were caught cheating, as they have to reset the achievements earned by cheating (if not, whole profile. can't remember which is which). I'd imagine it would have to be the same if this feature were to ever be implemented as well.

**Comment by luchaos** [comment]
I love points so much I'd completely invert it and make it opt-in instead. 🔥 

In any case - being untracked and requesting to be tracked again is subject to some form of a "cheat review". The full reset is just the easiest way to guarantee a user has no infringing unlocks.

I think this feature would have to go in hand with improved moderation features that allow for easy introspection, a "re-track" task queue to triage requests, etc. Turning that flag on and off in the database is easy. Keeping track of it being changed by whom, when, and why requires additional thought and quite a bit more work.

**Comment by LuisMayo** [comment]
> The original purpose was to basically shadowban cheaters, mainly so they can continue cheating etc without it actually affecting legit players. Not sure about it being a player-facing feature

That I didn't know, this suggestion makes less sense in that case. But I think there are a lot of people who care about achievements but not about competition and want to play co-op/whatever. Although I guess that kind of people would never get enough points to disrupt competition anyway

> Keeping track of it being changed by whom, when, and why requires additional thought and quite a bit more work.

That's what I was afraid of so yeah, no reversion would be possible then.

Taking your comments into account the suggestion may not be good enough to warrant development, that's for you to decide.

TBH I just discovered your project (awesome btw) and registered to play Silent Hill with my bf with achievements. Only to find later I was "breaking" the rules by playing two people in the same account since I didn't realize it was a competitive-oriented project.

Thanks!

**Comment by Sanaki** [comment]
While it's not manual, right now users can message RAdmin on-site with a request to be untracked. You're weclome to do so at your leisure.
